cmd/rm-main.go

```diff
@@ -129,7 +129,7 @@ type rmMessage struct {
 
 // Colorized message for console printing.
 func (r rmMessage) String() string {
-	return console.Colorize("Remove", fmt.Sprintf("Removing `%s`.", r.Key))
+	return console.Colorize("Remove", fmt.Sprintf("Attempting to remove `%s`.", r.Key))
```

Comment:
We should not add this message, we should print the message after successful remove @sinhaashish so the message should be "Removed %s"